js/components/Editor.js

```diff
@@ -124,12 +124,13 @@ const Editor = React.createClass({
 
   render() {
     return (
-      dom.div(
-        { className: "editor" },
-        dom.textarea({
-          ref: "editor",
-          defaultValue: "..."
-        })
+      dom.div({ className: "editor-container" },
```

Comment:
Are you sure you need a container? A previous commit made `.editor` have a width/height of 100% but it was removed for some reason. I think it was like this before:

```
.editor { flex: 1}

.CodeMirror { width: 100%; height: 100% }
```

You're supposed to give `.CodeMirror` a width/height if you want to tell CodeMirror what it should be. I think that should work and avoids a container element, but I could be wrong.